webapi/ticketstatus.go

```diff
@@ -34,12 +34,24 @@ func ticketStatus(c *gin.Context) {
 		return
 	}
 
+	// Get altSignAddress from database
+	altSignAddrData, err := db.AltSignAddrData(ticket.Hash)
+	if err != nil {
+		log.Errorf("db.AltSignAddrData error (ticketHash=%s): %v", ticket.Hash, err)
```

Comment:
Log line should be prefixed with `funcName`.

Reply:
An example where we already call `db.AltSignAddrData`

https://github.com/decred/vspd/blob/0fd96388cecd6506174ba9bc33d6e77d42a5fc90/webapi/setaltsignaddr.go#L57-L62